ballerina/Module.md

```diff
@@ -0,0 +1,426 @@
+   If you define the `connectionPool` field inline when creating the database client with the `sql:ConnectionPool` type,
+   an unsharable connection pool will be created.
+
+    ```ballerina
+    oracledb:Client|sql:Error dbClient = new (user = "adminUser", password = "adminPassword",
+                                           connectionPool = { maxOpenConnections: 5 });
+    ```
+
+3. Local, shareable connection pool
+
+   If you create a record of the`sql:ConnectionPool` type and reuse that in the configuration of multiple clients,
+   for each set of clients that connects to the same database instance with the same set of properties, a shared
+   connection pool will be created.
+
+    ```ballerina
+    sql:ConnectionPool connPool = {maxOpenConnections: 5};
+    
+    oracledb:Client|sql:Error dbClient1 =       
+                               new (user = "adminUser", password = "adminPassword",
+                               connectionPool = connPool);
+    oracledb:Client|sql:Error dbClient2 = 
+                               new (user = "adminUser", password = "adminPassword",
+                               connectionPool = connPool);
+    oracledb:Client|sql:Error dbClient3 = 
+                               new (user = "adminUser", password = "adminPassword",
+                               connectionPool = connPool);
+    ```
+
+For more details about each property, see the [`oracledb:Client`](https://docs.central.ballerina.io/ballerinax/oracledb/latest/clients/Client) constructor.
+
+The [oracledb:Client](https://docs.central.ballerina.io/ballerinax/oracledb/latest/clients/Client) references
+[sql:Client](https://docs.central.ballerina.io/ballerina/sql/latest/clients/Client) and all the operations
+defined by the `sql:Client` will be supported by the `oracledb:Client` as well.
+
+#### Closing the Client
+
+Once all the database operations are performed, you can close the database client you have created by invoking the `close()`
+operation. This will close the corresponding connection pool if it is not shared by any other database clients.
+
+```ballerina
+error? e = dbClient.close();
+```
+Or
+```ballerina
+check dbClient.close();
+```
+
+### Database Operations
+
+Once the client is created, the database operations can be executed through that client. This module defines the interface
+and common properties that are shared among multiple database clients.  It also supports querying, inserting, deleting,
+updating, and batch updating data. 
+
+#### Creating Tables
+
+This sample creates a table with two columns. One column is of the `int` type and the other is of the `varchar` type.
+The `CREATE` statement is executed via the `execute` remote function of the client.
+
+```ballerina
+// Create the ‘Students’ table with the  ‘id’, 'name', and ‘age’ fields.
+sql:ExecutionResult result = check dbClient->execute("CREATE TABLE student(id INT AUTO_INCREMENT, " +
+                         "age INT, name VARCHAR(255), PRIMARY KEY (id))");
```

Comment:
Don't use string queries. use parameterized queries

Reply:
check and correct all places

Reply:
corrected in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/27/commits/bcf2f27f652970e8e3740ebf430790b44d9c2978